app/models/manageiq/providers/embedded_ansible/automation_manager/job.rb

```diff
@@ -96,7 +98,7 @@ def retireable?
   # If extra_vars are passed through automate, all keys are considered as attributes and
   # converted to lower case. Need to convert them back to original definitions in the
   # job template through survey_spec or variables
-  def self.reconcile_extra_vars_keys(_template, options)
+  def self.reconcile_extra_vars_keys(_playbook, options)
```

Comment:
@lfu is this just returning back the options that were passed in

Reply:
Seems that is the case here.

Reply:
@lfu can we get rid of it

Reply:
If it is ok with @gmcculloug. I promised him this is a refactor PR.

Reply:
@lfu Since this is a private method it seems like a straight-forward change that would be fine for this PR.